Add support for falco on OpenShift 4.6 (#1354)
SumoLogic/sumologic-kubernetes-collection: deploy/docs/Troubleshoot_Collection.md

  #ebpf:
  #  enabled: true
```

### Falco

> Falco does not provide modules for all kernels. When there is missing Falco module Falco tries to build it.

Reviewer: ```suggestion
Falco does not provide modules for all kernels.
When Falco module is not available for particular kernel, Falco tries to build it.
```

Building of module requires `kernel-devel` package installed on nodes.

For OpenShift installation of `kernel-devel` on nodes is provided through MachineConfig used by
[Machine Config operator](https://github.com/openshift/machine-config-operator).
When update of machine configuration is needed machine is rebooted, please see
[documentation](https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfigDaemon.md#coordinating-updates).
The process of changing nodes configuration may require long time
during which Pods scheduled on unchanged nodes are in `Init` state.

> Configuration of node can be verified by following annotations:

Reviewer: ```suggestion
Node configuration can be verified by following annotations:
```

  #ebpf:
  #  enabled: true
```

### Falco

Falco does not provide modules for all kernels.
When Falco module is not available for particular kernel, Falco tries to build it.

> Building of module requires `kernel-devel` package installed on nodes.

Reviewer: ```suggestion
Building a module requires `kernel-devel` package installed on nodes.
```

Sorry missed that nit.

> For OpenShift installation of `kernel-devel` on nodes is provided through MachineConfig used by

Reviewer: ```suggestion
For OpenShift, installation of `kernel-devel` on nodes is provided through `MachineConfig` used by
```
